contents/handbook/growth/sales/professional-services.md

```diff
@@ -26,8 +26,60 @@ For ongoing training, this is something that we are solving for separately, but
+
+ - PostHog SDK implementation.
+ - User identification.
+ - Privacy controls.
+ - Sampling.
+ - Autocapture tuning.
+ - Custom event instrumentation.
+ - Identifying key insights/dashboards to be created.
+
+At the end of this session we should have a good plan and understanding of any onsite work which needs to take place, as well as who in the customer our engineer will be working with, and what level of access to customer systems they require.
+
+### Technical implementation
+
+Here we make sure that PostHog is correctly integrated into the codebase using one or more of our SDKs.  We should also:
+
+ - Implement any user identification or privacy controls that are required.
+ - Ensure they are only tracking the events which they need to (including implementing custom events where necessary).
+ - Integrate the first set of feature flags into their application.
+ - Set up the relevant dashboards and insights as documented in the scoping phase.
+ - Ensure that everything is set up according to our best practice [guidance](/handbook/cs-and-onboarding/health-checks).
+
+### Getting started training
+
+Once data is integrated, we should provide an intro to PostHog session to the customer to teach them the basics of how to use PostHog.  This will be tailored to their needs but should provide a baseline understanding of how to navigate the UI, where to find events, create insights, filter replays, etc.
+
+### Migration
+
+Whilst it's crucial to get live data flowing in to PostHog, the customer may also want to bring over historic data to PostHog from their previous tools.  This will normally be product analytics data, and we have both [managed](/docs/migrate/managed-migrations) and [manual](/docs/migrate/migrate-to-cloud) processes for this depending on the incumbent tool.
+
+> Longer term it's expected that forward deployed engineers will own the managed migration tools and also build out that capability.
+
+Once data is migrated, we may also need to implement dashboards and insights from the previous tool.  There's no automated way to do this currently, so we will need a login to the previous tool to understand and recreate the visualizations they need to move over.
+
+If the customer is replacing a feature flagging tool and has existing feature flags in place, we will need to migrate the flags in the codebase and ensure that the flags and targeting are set up correctly in PostHog.
+
+### Data connectivity
+
+This will normally involve the setup of [realtime](/docs/cdp/destinations) and [batch export](/docs/cdp/batch-exports) destinations for other tools in the customer's stack.  We'll need API keys for the relevant tools as well as an agreed criteria for choosing which events will go to which destination.
+
+This may also involve getting data into PostHog without using our SDKs, for example, by using the [Webhooks](/docs/cdp/sources/incoming-webhooks) or Data Warehouse [sources](/docs/cdp/sources).
+
+### SQL Query implementation
```

Comment:
Is there a risk this becomes an ongoing need from a customer where they expect us to write SQL for them?